### ISSUE TITLE
Wry – a web view library by Tauri

### ISSUE THREAD

**Issue by muuvmuuv** [vlang/vbrowser#8]
Would also be a nice to consider, it's written in Rust.

https://github.com/tauri-apps/wry

**Comment by JalonSolov** [comment]
That could wind up being a lot of dependencies.  And we would have to set up a system to build it all from source, generate all the variations of shared/static libraries we might need (since they don't - they only supply source), and figure out a way to distribute the correct one(s) for each system.

Oh, and write a V wrapper for all of it.

**Comment by alula** [comment]
It's not a browser engine but a wrapper for system provided ones - trivially doable in V

**Comment by JalonSolov** [comment]
If you can convert the rust code to V, please do.  :-)

**Comment by muuvmuuv** [comment]
@alula yes, right, read to many things at once today and mixed it with something else I was looking at. Could be a good starting point to look at.
